package.json

```diff
@@ -194,6 +195,9 @@
     "rn-nodeify": "^10.2.0",
     "yarn-deduplicate": "^2.1.1"
   },
+  "resolutions": {
```

Comment:
workaround for https://github.com/react-native-svg/react-native-svg/issues/591#issuecomment-437702637